Implemented vulkan instance

0xen opened the pull request:
Removed renderMesh from renderer
Added render to renderer

DontBelieveMe commented:
The code itself looks good - can you replace the tabs with 4 spaces, just so we're consistant.

0xen commented:
Added Vulkan Physical Device
Added SDL -> Vulkan surface code into VulkanWindow
Untabified all files
Removed central header file as with cmake settings, it picks up false positive circular dependency when they don't exist.

DontBelieveMe commented:
Awsome :D thanks for doing the tab/space thing. Merging.
